.github/workflows/link-check-deploy.yml

```diff
@@ -3,6 +3,4 @@ on:
   deployment_status:
 jobs:
   run:
-    uses: iterative/link-check/.github/workflows/link-check-deployment-status.yml@v0.13.0
-    with:
-      main: master
+    uses: iterative/link-check/.github/workflows/link-check-deployment-status.yml@v0.13.1
```

Comment:
#287

Reply:
can we just do `v0` rather than `v0.13.1` to make it auto-update? Or is there already some easy process to bump versions?